src/com/mendix/widget/carousel/components/__tests__/CarouselControl.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { ShallowWrapper, shallow } from "enzyme";
+import { DOM, createElement } from "react";
+
+import { CarouselControl, CarouselControlProps } from "../CarouselControl";
+
+describe("CarouselControl", () => {
+
+    let carouselControl: ShallowWrapper<CarouselControlProps, any>;
+    let clickCount = 0;
+    let onClick = () => clickCount++;
+
+    beforeEach(() => {
+        carouselControl = shallow(createElement(CarouselControl, { direction: "right", onClick }));
+    });
+
+    it("renders the structure correctly", () => {
+        expect(carouselControl).toBeElement(
+            DOM.div({ className: "widget-carousel-control right", onClick },
+                DOM.span({ className: "glyphicon glyphicon-chevron-right" })
+            ));
+    });
+
+    it("renders with the carousel-control css class", () => {
+        expect(carouselControl.hasClass("widget-carousel-control")).toBe(true);
```

Comment:
Use the correct matcher for it, it will produce better errors if the test fails. So `expect(...).tohaveClass("...")`.